must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -382,6 +382,10 @@ for ns in $namespaces; do
     ceph_daemon_log_collection() {
         dbglog "collecting Ceph daemon logs from node ${node}"
         oc rsync -n "${ns}" "$(oc get pods -n "${ns}" --no-headers | grep "${node//./}-debug" | awk '{print $1}')":/host/var/lib/rook/"${ns}"/log "${CEPH_DAEMON_LOG_OUTPUT_DIR}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}" --no-headers | grep "${node//./}-debug" | awk '{print $1}')":/host/sys/kernel/debug/ceph "${CEPH_LOG_OUTPUT_DIR}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}" --no-headers | grep "${node//./}-debug" | awk '{print $1}')":/host/sys/bus/rbd "${RBD_LOG_OUTPUT_DIR}"
```

Comment:
How does `oc rsync` interact with symlinks?  There is plenty of those under `/sys/bus/rbd` and we want that data too.

Could you please upload the resulting `libceph_${node}` and `rbd_${node}` directories somewhere after taking must-gather on a cluster with at least two active RBD PVs?